simtools/db_handler.py

```diff
@@ -590,53 +638,10 @@ def _getSiteParametersMongoDB(
             _parameters[parNow].pop('Parameter', None)
             _parameters[parNow].pop('Site', None)
             _parameters[parNow]['entryDate'] = ObjectId(post['_id']).generation_time
-            if _parameters[parNow]['File']:
```

Comment:
Do you take care of this somewhere else? Notice that for the sites parameters' it's done separately so when you get the site parameters you also need to read the associated files.
Maybe you took care of it, It's hard to tell from the review since I only see the changes here (and finding it in the code would be hard).

Reply:
Yes. This is done externally. The TelescopeModel, for example, adds the site parameters to the list of files to export.